### DISCUSSION TITLE
How DFTB+ manages atomic charges?

### DISCUSSION THREAD

**Discussion by gekbuccella** [dftbplus/dftbplus#1143] [Q&A]
Hi everyone, I'm asking for help about an issue to which I don't find any answer in the manual.
I'm interested in simulating a chemical system with excess (or lack) of charge on certain particular atoms.
The problem is that the after a single MD step (timestep=0.5 fs), the charge is already delocalized among all atoms in the cell. This occurs even if I'm simulating an ion approaching a distant surface.

Example:
I want to simulate the ion N(+) approaching a surface, so I set the initial charge of N atom = +1.000 :

InitialCharges = {
   AtomCharge = {
    Atoms = N
    ChargePerAtom = +1.0000
    } 
  }

therefore, I expect that at the beginning of the simulation the number of electrons associated with N would be 5-1=4.
Well, after a single MD step, its charge is already about 4.9, even if it is still 10 AA away from other atoms in the system with which it could interact.

My question is: how is this possible? is it something related to the way DFTB+ manages the charge distribution? Do I need a time-dependent calculation for this?
Any suggestion would be really appreciated
Many thanks

Giacomo

**Comment by aradi** [comment]
Answer was given [here](https://mailman.zfn.uni-bremen.de/pipermail/dftb-plus-user/2022/004014.html).

**Reply by rvipa-dtu** [reply]
this link is broken :(
